cocotb/log.py

```diff
@@ -173,86 +166,149 @@ class SimLogFormatter(logging.Formatter):
     """
 
     # Removes the arguments from the base class. Docstring needed to make
-    # sphinx happy.
+    # Sphinx happy.
     def __init__(self):
         """ Takes no arguments. """
         super().__init__()
 
     # Justify and truncate
     @staticmethod
     def ljust(string, chars):
+        """Left-justify and truncate *string* to *chars* length."""
+
         if len(string) > chars:
-            return ".." + string[(chars - 2) * -1:]
+            return "".join(["..", string[(chars - 2) * -1:]])
```

Comment:
The truncation should be more configurable, for instance, we could use the ellipsis … to save one character.